api/VERSION

```diff
@@ -0,0 +1 @@
+v2.1-rc1
```

Comment:
🤔 Do we have to modify this file before releasing it? The release manager may forget this point sometimes. We should update docs or add some tips somewhere 😂

Reply:
Yeah. It's important. I had checked the apisix project. It needs to update `version.lua` for every release.
The same as apisix-dashboard. 

Maybe we can add a checklist to the PR template?

Reply:
Got it, we may have this tip to release guide 😃